README.adoc

```diff
@@ -32,25 +32,25 @@ Here is a quick teaser of a complete Spring Boot application in Java:
 	@SpringBootApplication
 	public class Example {
 
-		@RequestMapping("/")
-		String home() {
-			return "Hello World!";
-		}
+	    @RequestMapping("/")
+	    String home() {
+	        return "Hello World!";
+	    }
 
-		public static void main(String[] args) {
-			SpringApplication.run(Example.class, args);
-		}
+	    public static void main(String[] args) {
+	        SpringApplication.run(Example.class, args);
+	    }
 
```

Comment:
We prefer tabs as they're [more accessible](https://dev.to/alexandersandberg/why-we-should-default-to-tabs-instead-of-spaces-for-an-accessible-first-environment-101f).

Reply:
Ok, I see.